docs/en/pact-functions.md

```diff
@@ -5,14 +5,14 @@
 
 Constant denoting the ASCII charset
 
-Constant: 
+Constant:
```

Comment:
OH I know what it is....

My editor is deleting trailing space whenever I modify something like `pact-functions` directly.